t/selenium/release-editor/The_Downward_Spiral.json5

```diff
@@ -447,18 +447,32 @@
       target: "xpath=//a[@href='#recordings']",
       value: '',
     },
-    // Set the first track to an existing recording, pasting an MBID.
+    // Open the recording bubble for the first (non-pregap) track.
```

Comment:
The `[2]` subscript below (1-based because XPath :-/) always throws me for a loop, so I'm updating this comment to describe what I think is going on here.